crypto/keys/src/ed25519.rs

```diff
@@ -356,6 +386,104 @@ impl Verifier<Ed25519Signature> for Ed25519Pair {
     }
 }
 
+/// An Ed25519 signature.
+#[derive(Copy, Clone, Debug, Deserialize, Serialize)]
+pub struct Ed25519Signature(Signature);
```

Comment:
If we want a Signature that can be included in structs that `derive(Message)`, this is the price for it :/